From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@YoungjaeKim__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

PRmerger commented:
@YoungjaeKim : Thanks for your contribution to the Azure documentation! The author, @azdocprs, has been notified to review your proposed change.

YoungjaeKim commented:
PS. Sorry for the bad English in this PR as I'm not good at English. I appreciate you to revise whole sentence.

jomolnar commented:
@BryanLa Hi Bryan. Please review the proposed changes. Thanks.

BryanLa commented:
@jomolnar - Hi Joseph - I should probably defer to @tysonn on this one, mainly because he owns the Contributor Guide. Also, I'm wondering if something like this would be better in one of the PR related topics, ie: contributor-guide-pull-request-comments.md or contributor-guide-pull-request-etiquette.md ?

BryanLa commented:
Ping for @tysonn

tysonn commented:
@YoungjaeKim - Could you check out my PR to your fork? https://github.com/YoungjaeKim/azure-content/pull/3. I submitted some revisions for you to review and merge.

tysonn commented:
@YoungjaeKim - Merged, thank you!

itechedit committed:
Merge pull request #37005 from mattbriggs/mdb_20180404_storageupdate

updating storage topic

ktoliver committed:
Merge pull request #38094 from v-nagta/jira52metadataurl

User Story 1238736: SaaS App Tutorial: Microsoft Azure Active Directo…

ktoliver committed:
Merge pull request #38098 from v-viinde/yardionemetadataupdate

USER STORY 1238749 SaaS App Tutorial: YardiOne Metadata URL Update

jmprieur committed:
Redirection of
articles\active-directory\develop\active-directory-v2-devquickstarts-wpf.md
to
https://docs.microsoft.com/en-us/azure/active-directory/develop/guidedsetups/active-directory-windesktop

ktoliver committed:
Merge pull request #38099 from v-viinde/xaitportermetadataupdate

USER STORY 1238748 SaaS App Tutorial: XaitPorter Metadata URL Update

Commit:
fix for git issue 7189

itechedit committed:
Merge pull request #38404 from sujayt/master

com-error

ktoliver committed:
Merge pull request #38100 from v-viinde/trisotechmetadataupdate

USER STORY 1238746 SaaS App Tutorial: Trisotech Digital Enterprise Se…

tynevi committed:
Update .acrolinx-config.edn

ktoliver committed:
Merge pull request #38101 from v-nagta/ciscocloudlockupdate

User Story 1237060: SaaS App Tutorial: supplychain.cisco.com (Cisco C…

ktoliver committed:
Merge pull request #38102 from v-viinde/springerlinkmetadataupdate

USER STORY 1238745 SaaS App Tutorial: Springer Link Metadata URL Update

ktoliver committed:
Merge pull request #37555 from v-nagta/secretserver

USER STORY 1232910 SaaS App Tutorial: Secret Server (On-Premises)

cjgronlund committed:
breadcrumb fixes - cognitive services

ktoliver committed:
Merge pull request #38103 from v-viinde/spaciometadataupdate

USER STORY 1238743 SaaS App Tutorial: Spacio Metadata URL Update

Ja-Dunn committed:
Merge pull request #38406 from jimdial/Remove-NRP

Removed NRP article

PRMerger12 committed:
Merge pull request #38423 from davidmu1/gitissue7189

fix for git issue 7189

Commit:
Task 985: Update option 2 example

Ja-Dunn committed:
Merge pull request #38106 from v-nagta/safetynet

User Story 1237078: SaaS App Tutorial: SafetyNet Metadata URL Update

ktoliver committed:
Merge pull request #38104 from v-viinde/signalfxmetadataupdate

USER STORY 1238742 SaaS App Tutorial: SignalFx Metadata URL Update

jmprieur committed:
fixing the redirection.json

mimig1 committed:
Cosmos DB: Updated screenshots and refreshed tutorial

mimig1 committed:
date update

itechedit committed:
Merge pull request #37438 from cjgronlund/cog-serv-hub

Cog serv hub

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

Ja-Dunn committed:
Merge pull request #38105 from v-nagta/servicenowurl

User Story 1237080: SaaS App Tutorial: Servicenow Metadata URL Update

Commit:
upgrade acrolinx score to above 90

ktoliver committed:
Merge pull request #38407 from cephalin/issue6385

#6385

mimig1 committed:
removed "will"

Commit:
Removed file.

billmath committed:
adding toc entry

mimig1 committed:
QA pass

Commit:
fix for git issue 7283

Commit:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr

v-alje committed:
Merge pull request #38431 from MicrosoftDocs/FromPublicRepo

From public repo

itechedit committed:
Merge pull request #38429 from dksimpson/dean-985

Task 985: Update option 2 example

ktoliver committed:
Merge pull request #38426 from jmprieur/cleanup_388667

Redirection of active-directory-v2-devquickstarts-wpf.md to a guided setup

Commit:
updating formatting, tabbed content which isn't code

HeidiSteen committed:
revert a metadata update + remove Chrome reference to Postman

ktoliver committed:
Merge pull request #36964 from kgremban/mar30-iossdk

Quickstart send telemetry from iOS device

ktoliver committed:
Merge pull request #38405 from jimdial/vnet-classic

Virtual network classic author updates

ktoliver committed:
Merge pull request #38316 from billmath/guests

updating

mimig1 committed:
Cosmos DB: Removing experimental files that were never used

Commit:
fixed formatting around node-pending-pipe-instances

dlepow committed:
Another screenshot update

ktoliver committed:
Merge pull request #38428 from mimig1/04192018powerbi

Cosmos DB: Updated screenshots and refreshed tutorial

ktoliver committed:
Merge pull request #38433 from davidmu1/gitissue7283

fix for git issue 7283

mimig1 committed:
Cosmos DB: Applying contextual TOC links

goraco committed:
Update sap-high-availability-infrastructure-wsfc-file-share.md

fix the link

MicrosoftGuyJFlo committed:
Merge branch 'master' into AzureADAuthMoves5

rwike77 committed:
Merge branch 'https2' of https://github.com/rwike77/azure-docs-pr into https2

rwike77 committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into https2

rwike77 committed:
updated kestrel configuration

Commit:
luis01

v-shils committed:
Merge pull request #38437 from HeidiSteen/heidist-master

Azure Search: Revert a metadata update + remove Chrome reference to Postman

ktoliver committed:
Merge pull request #38323 from dlepow/fix

Batch AD: Freshness and Python SP sample

Commit:
Merge pull request #37980 from paulgarn/aad-tokenlifetime-dep

Token Lifetime Policy: Update to top note to warn of deprecation

neilpeterson committed:
fixes MicrosoftDocs/azure-docs#7446

rwike77 committed:
added info on exporting cert

mimig1 committed:
updated all shared docs to use Cosmos DB as contextual TOC

michhar committed:
Removed the newline throwing Note off

Had extra new line in Note - removed

Commit:
updating acrolinx after formatting fix

PRMerger10 committed:
Merge pull request #38448 from neilpeterson/aks-issue-7446

fixes MicrosoftDocs/azure-docs#7446

itechedit committed:
Merge pull request #38438 from mimig1/04192018exp

Cosmos DB: Removing experimental files that were never used

anzaman committed:
Update vpn-gateway-sample-create-vpn-gateway-powershell.md

fixing a typo

anzaman committed:
Update vpn-gateway-sample-point-to-site-radius-authentication-powershell.md

anzaman committed:
Update vpn-gateway-sample-point-to-site-certificate-authentication-powershell.md

anzaman committed:
Update vpn-gateway-sample-site-to-site-powershell.md

CarlRabeler committed:
[sql db] redirect

Brenduns committed:
revision to 1802 post update steps.

Commit:
ran acrolinx, added alt tags

PRMerger18 committed:
Merge pull request #38450 from anzaman/patch-3

Update vpn-gateway-sample-create-vpn-gateway-powershell.md

PRMerger18 committed:
Merge pull request #38451 from anzaman/patch-4

Update vpn-gateway-sample-point-to-site-radius-authentication-powersh…

PRMerger18 committed:
Merge pull request #38452 from anzaman/patch-5

Update vpn-gateway-sample-point-to-site-certificate-authentication-po…

Commit:
quickfix23

Commit:
Minor updates to the tutorial

PRMerger19 committed:
Merge pull request #38454 from anzaman/patch-6

Update vpn-gateway-sample-site-to-site-powershell.md

barbkess committed:
fix merge

barbkess committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into ant

barbkess committed:
updates

elisolMS committed:
updated based on feedback

Commit:
quickfix89

itechedit committed:
Merge pull request #34547 from SyntaxC4/patch-6

Updating metadata

Brenduns committed:
same

Commit:
Merge pull request #38457 from MarkusVi/quickfix23

quickfix23

Brenduns committed:
same

bandersmsft committed:
azure policy - added clarity to create-manage-policy

MicrosoftGuyJFlo committed:
[Azure AD] SSPR Fix for comment in issue 7424

itechedit committed:
Merge pull request #38335 from SyntaxC4-MSFT/cfowler-node-troubleshoot

Updating Blog Url

v-shils committed:
Merge pull request #38416 from michhar/patch-3

More Python instructions to linux-dsvm-walkthrough

v-alje committed:
master redirect

rjagiewich committed:
Merge pull request #37792 from rwike77/https2

adding article to enable https in asp.net core

PRMerger12 committed:
Merge pull request #38459 from Brenduns/quick-fix

Quick fix

itechedit committed:
Merge pull request #35356 from dlepow/batchmonitor

Batch monitoring: new and updated content

PRMerger14 committed:
Merge pull request #38460 from MarkusVi/quickfix89

quickfix89

twooley committed:
[Azure AD B2B] Updates to allow deny list article

PRMerger14 committed:
Merge pull request #38462 from bandersmsft/master

azure policy - added clarity to create-manage-policy

v-alje committed:
cognitive-services

PRMerger15 committed:
Merge pull request #38463 from MicrosoftGuyJFlo/Issue7424

[Azure AD] SSPR Fix for comment in issue 7424

Commit:
title change, acrolinx, md lint

cherylmc committed:
update

cherylmc committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into ov

v-alje committed:
Merge pull request #38465 from MicrosoftDocs/redirection-fix

redirection fix

Commit:
Merge pull request #38441 from goraco/master

fix the link to prerequisite document

mimckitt committed:
Fixes MicrosoftDocs/azure-docs#7435

Get-AzureRMVMImage -Location $locName -Publisher $pubName -Offer $offerName -Sku skuName | Select Version
should read:
Get-AzureRMVMImage -Location $locName -Publisher $pubName -Offer $offerName -Sku $skuName | Select Version

twooley committed:
[Azure AD B2B] Updates to allow deny list article

elisolMS committed:
Merge branch 'update-how-apps-are-added' into patch-2

Ja-Dunn committed:
Merge pull request #38455 from CarlRabeler/master

[sql db] redirect

rjagiewich committed:
Update .openpublishing.redirection.json

Fixed typo in a redirect at line 18,810

Commit:
Merge pull request #38470 from MicahMcKittrick-MSFT/patch-20

Fixes MicrosoftDocs/azure-docs#7435

Ja-Dunn committed:
Merge pull request #38461 from dan-wesley/dawesl-quick-windows-portal-apr19

dawesl quick windows portal apr19

Commit:
toc item fix

elisolMS committed:
Update active-directory-how-applications-are-added.md

modified a couple changes

elisolMS committed:
Merge pull request #1 from CelesteDG/patch-2

Editorial changes and voice and tone updates

Commit:
automation : new parameter for script

rjagiewich committed:
Merge pull request #38467 from twooley/twb2badlistupdate

[Azure AD B2B] Updates to allow deny list article

tynevi committed:
Merge pull request #38374 from MicrosoftGuyJFlo/AzureADAuthMoves5

[Azure AD] Authentication MVC File Moves - Part 5

Commit:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into dawesl-quick-windows-powershell-apr19

elisolMS committed:
Update active-directory-how-applications-are-added.md

fixed term usage

Commit:
updates

itechedit committed:
Merge pull request #35152 from barbkess/ant

Ant

Ja-Dunn committed:
Merge pull request #38432 from cjgronlund/cog-serv-bc

breadcrumb fixes - cognitive services

Ja-Dunn committed:
Merge pull request #38229 from MarkusVi/luis01

luis01

v-geberr committed:
Carol's suggestion

barbkess committed:
author fixes

Commit:
title change, changes as per acrolinx and linter

GitHubber17 committed:
Merge pull request #38329 from elisolMS/update-how-apps-are-added

updated how apps are added doc

cherylmc committed:
update

dlepow committed:
Batch rendering: update for new Windows image

barclayn committed:
deleting the old cli article and redirecting to the cli2 article

JasonWHowell committed:
Update stream-analytics-define-inputs.md

JasonWHowell committed:
Update stream-analytics-define-inputs.md

v-geberr committed:
edits

Commit:
automation : rbac updates

JasonWHowell committed:
Update stream-analytics-define-inputs.md

mimig1 committed:
Fixing issues

PRMerger17 committed:
Merge pull request #38477 from dlepow/render

Batch rendering: update for new Windows image

LuisBosquez committed:
Fixing version for Gremlin.Net

LuisBosquez committed:
Revert "Added missing supported Gremlin steps"

This reverts commit 96ed5936e5ca707b258bb2e6a54c65eda95cef31.

LuisBosquez committed:
Revert "Added missing supported Gremlin steps"

This reverts commit 0484ec406e7093b6b377cfbbe7ce526f1b1ced09.

Commit:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into dawesl-quick-windows-cli-apr19

cherylmc committed:
update

Commit:
update metadata date

JasonWHowell committed:
Update stream-analytics-define-inputs.md

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

cherylmc committed:
update

ktoliver committed:
Merge pull request #38480 from barclayn/keyvault-fix02

deleting the old cli article and redirecting to the cli2 article

cephalin committed:
changed date formatting

ktoliver committed:
Merge pull request #37776 from cephalin/msi

add MSI tutorial

Commit:
Edits after PR 37857.

ktoliver committed:
Merge pull request #38484 from v-geberr/0419-phraselist-based-on-public-pr

0419 phraselist based on public pr

ktoliver committed:
Merge pull request #38488 from dan-wesley/dawesl-quick-windows-cli-apr19

dawesl quick windows cli apr19

ktoliver committed:
Merge pull request #38478 from JasonWHowell/patch-2

Update stream-analytics-define-inputs.md

cherylmc committed:
Update vpn-gateway-table-site-to-site-include.md

v-alje committed:
Merge pull request #38490 from MicrosoftDocs/FromPublicRepo

From public repo

cherylmc committed:
update

cherylmc committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into ov

SnehaGunda committed:
Updating partition doc

SnehaGunda committed:
Minor update

MicrosoftGuyJFlo committed:
[Azure AD] Authentication MVC File Moves - Part 6

PRMerger20 committed:
Merge pull request #38492 from MightyPen/edit-after-pr37857-a

Edits after PR 37857.

cherylmc committed:
update

Commit:
Merge pull request #38475 from barbkess/master

author fixes

barbkess committed:
metadata updates

twooley committed:
[Azure AD B2B] New hybrid org articles

PRMerger10 committed:
Merge pull request #38495 from SnehaGunda/cdbthroughput

Updating partition data info

MicrosoftGuyJFlo committed:
[Azure AD] Authentication MVC File Moves - Part 6

cherylmc committed:
update

PRMerger13 committed:
Merge pull request #38458 from raymondlaghaeian/patch-23

Minor updates to the tutorial

ktoliver committed:
Merge pull request #38350 from barbkess/musher

Add polling example

tfosmark committed:
Merge pull request #38489 from cherylmc/table

update

LuisBosquez committed:
Merge branch 'master' into version-fix

MashaMSFT committed:
added some clarity around oms, and diff between availability options

ajlam committed:
Fix mysql and pg create app services

ajlam committed:
remove conflict

ccompy committed:
clarified around IKEv2

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPrivateRepo

PRMerger17 committed:
Merge pull request #38472 from georgewallace/autohybrid

automation : new parameter for script

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo

MashaMSFT committed:
added some clarity

tynevi committed:
Merge pull request #38496 from MicrosoftGuyJFlo/AzureADAuthMoves06

[Azure AD] Authentication MVC File Moves - Part 6

Commit:
Merge pull request #38487 from LuisBosquez/version-fix

Version fix for Gremlin.Net

ghogen committed:
vs-azure-tools metadata updates

ghogen committed:
Merge branch 'master' of github.com:MicrosoftDocs/azure-docs-pr into metadata-vs-azure-tools

ghogen committed:
Update metadata on vs-azure-tools-cloud-service-publish-set-up-required-services-in-visual-studio.md

MashaMSFT committed:
changed "quickstart tutorial" to "quickstart"

PRMerger13 committed:
Merge pull request #38502 from ajlam/master

Fix mysql and pg create app services

ghogen committed:
metadata for vs-azure-tools-migrate-publish-web-app-to-cloud-service.md

MashaMSFT committed:
deleted some extraneous information

MashaMSFT committed:
made some small editorial fixes

Commit:
Merge pull request #38503 from ccompy/apr4

Added comment around disabling IKEV2

MashaMSFT committed:
made some editorial fixes

MashaMSFT committed:
changed some things for additional clarity

GitHubber17 committed:
Merge pull request #37778 from jeffgilb/413

Updated landing pages for new docset

GitHubber17 committed:
Merge pull request #38132 from cmcclister/1234200

Update to Security Information landing page (v2)

GitHubber17 committed:
Merge pull request #38367 from ghogen/metadata

Metadata changes in AD Connected Service topics

GitHubber17 committed:
Merge pull request #38380 from ghogen/metadata4

VS Storage Connected Service metadata updates

GitHubber17 committed:
Merge pull request #38508 from ghogen/metadata-vs-azure-tools

Metadata vs azure tools

GitHubber17 committed:
Merge pull request #38482 from georgewallace/autorbac

automation : rbac updates

Commit:
automation: correcting incorrect bullets

Commit:
removed bullets

PRMerger19 committed:
Merge pull request #38526 from georgewallace/autorbac

automation: correcting incorrect bullets

v-jasowells committed:
Update csharp.md

v-jasowells committed:
Update java.md

v-jasowells committed:
Update nodejs.md

v-jasowells committed:
Update php.md

v-jasowells committed:
Update python.md

v-jasowells committed:
Update ruby.md

Commit:
format fix to index page

PRMerger16 committed:
Merge pull request #38529 from v-jaswel/Translator-fix-quickstart-links

Translator fix quickstart links

linda33wj committed:
Add service URL property for Dynamics connector

Commit:
Merge pull request #38536 from linda33wj/master

Add service URL property for Dynamics connector

cephalin committed:
jelle feedback

mrbullwinkle committed:
Modify TOC to add link to Azure Funcitons monitoring

itechedit committed:
Merge pull request #37986 from cephalin/outbound

add ip address article

twooley committed:
[Azure AD B2B] New hybrid org articles

mrbullwinkle committed:
Changes to address GitHub Issue 6265

mrbullwinkle committed:
spelling error

mimig1 committed:
Removed reference link for Stream Analytics to keep pointer to specific section

PRMerger17 committed:
Merge pull request #38546 from mrbullwinkle/mrb_04_20_2018_analytics

Changes to address GitHub Issue 6265

PRMerger18 committed:
Merge pull request #38511 from MashaMSFT/20180419_sqldbservers

deleted some extraneous information

PRMerger19 committed:
Merge pull request #38512 from MashaMSFT/20180419_sqldbfixe

made some small editorial fixes

Commit:
Merge pull request #37026 from carolinacmoravia/patch-663

M24915: Error: "create the deploy" vs "create and deploy"

Brenduns committed:
Storage quota update - and cross link article

cherylmc committed:
update

cherylmc committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into ov

Commit:
Merge pull request #38505 from MashaMSFT/20180419_sqldbpaasvsiaas

added some clarity

Commit:
Merge pull request #38501 from MashaMSFT/20180419_techoverview

added some clarity around oms, and diff between availability options

Brenduns committed:
Storage quota update - and cross link article

Commit:
Merge pull request #38518 from MashaMSFT/20180419_sqldbfixes2

made some editorial fixes

Commit:
Merge pull request #38521 from MashaMSFT/20180419_sqldbfixes3

changed some things for additional clarity

eross-msft committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into lizross-backup-recovery-msauth

GitHubber17 committed:
Merge pull request #38442 from mimig1/04192018contextualtoc

Cosmos DB: Applying contextual TOC links

itechedit committed:
Merge pull request #38498 from twooley/twb2bhybridorgdocsfinal

[Azure AD B2B] New hybrid org articles

dominicbetts committed:
Address PR feedback

tfitzmac committed:
updated move

GitHubber17 committed:
Merge pull request #38506 from barbkess/master

metadata updates

Brenduns committed:
Storage quota update - and cross link article

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

itechedit committed:
Update sql-database-connect-query.md

itechedit committed:
Merge pull request #38473 from cherylmc/ov

overview doc

rjagiewich committed:
Merge pull request #38530 from cjgronlund/cog-serv-fix2

format fix to index page

GitHubber17 committed:
Merge pull request #38539 from mrbullwinkle/mrb_04_20_2018_az_functions

Modify TOC to add link to Azure Functions monitoring

PRMerger18 committed:
Merge pull request #38552 from tfitzmac/0420move

updated move

itechedit committed:
Merge pull request #38510 from MashaMSFT/20180419_qsfix

changed "quickstart tutorial" to "quickstart"

rjagiewich committed:
Merge pull request #38401 from dominicbetts/suite-cf-hmi-updates

Updates for HMI

v-alje committed:
Merge pull request #38553 from MicrosoftDocs/FromPublicRepo

From public repo

GitHubber17 committed:
Merge pull request #38550 from Brenduns/quick-fix

Quick fix

rjagiewich committed:
Merge pull request #35936 from eross-msft/lizross-backup-recovery-msauth

New end-user content for backup and recovery in Microsoft Authenticator app

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPrivateRepo

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
